client/src/components/Container.jsx

```diff
@@ -9,16 +10,31 @@ class Container extends React.Component {
   constructor(props) {
     super(props);
     this.toggleModal = this.toggleModal.bind(this);
+    this.getReviewInfo =this.getReviewInfo.bind(this);
     this.state = {
-      showModal: false
+      showModal: false,
+      reviews: []
     };
   }
 
+  componentDidMount() {
+    this.getReviewInfo(1);
+  }
+
   toggleModal() {
     this.setState({
       showModal: !this.state.showModal
     });
   }
+
+  getReviewInfo(storeId) {
+    axios.get(`/stores/${storeId}/reviews`)
```

Comment:
Is axios easier to use than jquery?

Reply:
I don't have too much of an issue with jquery but I thought I'd give axios a try. It was pretty easy!